.github/pull_request_template.md

```diff
@@ -1,11 +1,33 @@
-## Thank you for your contribution! 
+<!-- This template is **just a guide**, delete any and all parts which you don't need! -->
 
-Please add one or two sentences _why_ this PR was made, and optionally what it adds/changes…
+## 📝 Docs
 
-## Checklist
+- [ ] Did you update the [CHANGELOG.md](https://github.com/criticalmaps/criticalmaps-ios/blob/master/CHANGELOG.md)?
 
-### Before merging the PR
+## 📲 What
 
-- [ ] If applicable, does the PR contain enough unit / UI tests?
-- [ ] If applicable, did you add a before / after screenshot of the feature?
-- [ ] Did you update the [CHANGELOG.md](https://github.com/criticalmaps/criticalmaps-ios/blob/master/CHANGELOG.md)?
+A description of the change.
+
+## 🤔 Why
+
+Some background context on why the change is needed.
+
+## 👀 See
+
+Screenshots, external resources?
+
+| Before 🐛 | After 🦋 |
+| --- | --- |
+|  |  |
+
+## ♿️ Accessibility 
+
+- [ ] Tap targets use minimum of 44x44 pts dimensions
+- [ ] Works with VoiceOver
+- [ ] Supports Dynamic Type 
+
+## ⏰ TODO
+
+- [ ] Got planned refactors to come
+- [ ] Will add more tests
+- [ ] Need feedback on a design
```

Comment:
Should you check the field when there are open refactors or should it be unchecked because the refactors are still open? These three points would confuse me a bit.. 🤔

Reply:
I would check them if for example there is something I want to improve on an implementation but it would bloat the PR or if I want to add more tests

Reply:
As a way to say not done yet I think

Reply:
In my opinion, I would leave these points out. I think, it wouldn't be very often that we would actually have to use them.

Reply:
I removed them for now 👍